circleguard/loadable.py

```diff
@@ -706,3 +706,317 @@ def load(self, loader, cache):
         Replay.__init__(self, loaded.timestamp, map_id, loaded.player_name, user_id, ModCombination(loaded.mod_combination),
                         loaded.replay_id, loaded.play_data, self.weight)
         self.log.log(TRACE, "Finished loading %s", self)
+
+
+class ReplayModified(Replay):
```

Comment:
Modifying the `Replay` using the new methods only makes sense in the context of comparison,
so it may be more suitable to use `copy.deepcopy` and move these methods to a helper class or `Comparer`